light/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema":"../assetlist.schema.json",
+    "chain_name":"light",
```

Comment:
It sounds like this isn't its own chain, but rather a token on lumenx, in which case please add this to the assetlist there instead.

Reply:
It is a juno-chain token and has no chain (neither genesis not validators). Should I include it below lumen assetlist?

Reply:
if it's a juno token then it should go in Juno's assetlist. Why would it go in Lumen's assetlist?